app/views/articles/show.html.erb

```diff
@@ -103,73 +103,64 @@
+                      <a href="<%= @article.canonical_url || @article.feed_source_url %>" style="color:#1395b8"><%= get_host_without_www(@article.canonical_url || @article.feed_source_url) %></a>
+                      <% if @article.crossposted_at %>
+                        on
+                        <%= local_date(@article.originally_published_at || @article.published_at, show_year: (@article.originally_published_at || @article.published_at).year != Time.current.year) %>
+                      <% end %>
+                    <% end %>
+                  </p>
+                </div>
+              </div>
+            </div>
+
+            <h1 class="fs-3xl m:fs-4xl l:fs-5xl fw-bold s:fw-heavy lh-tight mb-2 <%= @article.title_length_classification %>">
               <% if @article.search_optimized_title_preamble.present? && !user_signed_in? %>
                 <span class="fs-xl color-base-70 block"><%= @article.search_optimized_title_preamble %></span>
               <% end %>
               <%= @article.title %>
             </h1>
 
             <% cache("main-article-tags-#{@article.cached_tag_list}", expires_in: 30.hours) do %>
-              <div class="mb-4 spec__tags">
+              <div class="spec__tags color-base-60">
                 <% @article.cached_tag_list_array.each do |tag| %>
-                  <% tag_hash = tag_colors(tag) %>
-                  <% styles = "" %>
-                  <% if tag_hash[:background] %>
-                    <% styles = "background-color:#{tag_hash[:background]};" %>
-                  <% end %>
-                  <% if tag_hash[:color] %>
-                    <% styles = "#{styles}color:#{tag_hash[:color]};" %>
-                  <% end %>
-                  <a class="crayons-tag mr-1" href="/t/<%= tag %>" style="<%= styles %>"><span class="crayons-tag__prefix">#</span><%= tag %></a>
+                  <a href="/t/<%= tag %>" class="crayons-link crayons-link--secondary"><%= tag %></a><%= "," unless tag == @article.cached_tag_list_array.last %>
                 <% end %>
               </div>
             <% end %>
-
-            <div class="crayons-article__subheader">
-              <a href="/<%= @user.username %>" class="flex items-center mr-2 m:mr-1 mb-4 s:mb-0 fw-medium crayons-link">
-                <span class="crayons-avatar crayons-avatar--l mr-2 m:mr-1"><img class="crayons-avatar__image" src="<%= Images::Profile.call(@user.profile_image_url, length: 50) %>" alt="" /></span>
-                <span class="block m:hidden"><%= @user.name %></span>
-              </a>
-              <div class="profile-preview-card relative mr-2 mb-4 s:mb-0 fw-medium">
-                <button id="profile-preview-trigger" class="profile-preview-card__trigger crayons-btn crayons-btn--ghost px-2 hidden m:block" aria-label="<%= @user.name %> profile details"><%= @user.name %></button>
-                <%= render "/shared/profile_preview_card", actor: @user, id: "profile-preview-content" %>
-              </div>
```

Comment:
I intentionally removed card previews as we have exactly the same thing in the right sidebar of article view - and that itself is sticky anyways 🤷

Reply:
I'm not sure if unconditionally removing it is really the best way forward. Maybe double-check with @aitchiss because IIRC we added it for viewport sizes where the right sidebar moves to the bottom.

Reply:
There's a breakpoint at which we currently have preview card in the byline, and the sidebar at the bottom of the page (basically everything between landscape mobile and desktop size screen). For portrait mobile screen sizes we already don't show the preview card, even though in this view the sidebar isn't really visible (it's at the very bottom of the article) 🤷‍♀️ 

The other situation it would be relevant for is on a post published under an organization - the sidebar contains the org details, whereas this is always specific to the author of the post rather than the org. 

I don't have any strong feelings either way about removing it here. The only reason I can see to keep it is for the organization posts and also just so there's more consistency in user name === preview card available.

Reply:
Just a heads up if we are removing it totally from here - we also need to remove the initialization in articlePage.jsx:

https://github.com/forem/forem/blob/e1606bcb0cf56deddc14682a70ea6d47998f0797/app/javascript/packs/articlePage.jsx#L126-L164

(this is what's causing the Cypress test failures at the moment)

Reply:
> The only reason I can see to keep it is for the organization posts

What if for org posts we would extend the sidebar box to contain user's info too (which honestly I think we should do anyway..)?

Reply:
> What if for org posts we would extend the sidebar box to contain user's info too (which honestly I think we should do anyway..)?

Yeah I think that would make sense 👍

Reply:
Would you mind if I tackle this in a separate PR? This might be slightly bigger change and I don't want to enlarge this PR. But I also don't want this to be a blocker..

> this is what's causing the Cypress test failures at the moment

BTW. I think I fixed that but still having issues with build... Would you mind giving a look @aitchiss ?

Reply:
Yeah for sure, I don't think it needs to be part of this PR. I'll take a look at the build 🙂